### PR TITLE
This markup shall not stand: part 4

### PR DESCRIPTION
**Pull request by CaseyCarter**

### CODE REVIEW

File: iterators.tex [hunk 1]
```diff
@@ -2991,41 +2986,39 @@
...
-    @\changed{typedef Container}{using}@ container_type@\added{ = Container}@;
-    @\added{using difference_type = ptrdiff_t;}@
-    @\added{insert_iterator();}@
-    insert_iterator(Container& x, typename Container::iterator i);
-    insert_iterator@\removed{<Container>}@&
+    using container_type = Container;
+    using difference_type = ptrdiff_t;
+
+    insert_iterator();
+    insert_iterator(Container& x, iterator_t<Container> i);
```
Comment (CaseyCarter) [inline iterators.tex]: I'm editorially interpreting that the intent of the change from `typename Container::iterator` to `iterator_t<Container>` was to make that change throughout `insert_iterator`, and not only to the few places that were actually changed,

Comment (ericniebler) [inline iterators.tex]: Actually, it was to brink the text body in line with a change that had already been made in the synopsis. I held off on the the other edits until LWG approved the change.

File: iterators.tex [hunk 2]
```diff
@@ -3777,7 +3695,7 @@
 is a \tcode{constexpr} constructor.
 \end{itemdescr}
 
-\indexlibrary{\idxcode{move_iterator}!constructor}%
+\indexlibrary{\idxcode{move_sentinel}!constructor}%
```
Comment (ericniebler) [inline iterators.tex]: Nice catch.

**Comment by ericniebler** [comment]
Fixes #210

File: iterators.tex [hunk 3]
```diff
@@ -3064,13 +3053,13 @@
 
 \indexlibrary{\idxcode{operator=}!\idxcode{insert_iterator}}%
 \begin{itemdecl}
-insert_iterator@\removed{<Container>}@&
+insert_iterator&
   operator=(const typename Container::value_type& value);
```
Comment (CaseyCarter) [inline iterators.tex]: I think I can call this editorial with a straight face.

**Pull request merged**